core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/download-requirements.sh

```diff
@@ -125,7 +126,11 @@ echol "Packages to be downloaded:"
 cat -n "${deplist}"
 
 # download dependencies (apt-get sandboxing warning when running as root are harmless)
-cd $dst_dir_packages && xargs --no-run-if-empty --arg-file=${deplist} --delimiter='\n' ${download_cmd} | tee -a ${logfile}
+export -f run_cmd_with_retries
+export -f run_cmd
+export -f print_array_as_shell_escaped_string
+export -f get_shell_escaped_array
```

Comment:
Do we need these exports?
These functions are sourced in line 35:
```
# source common functions
. "${script_path}/common.sh"
```

Reply:
yup, this is because of using xargs so it works in separate context